mods/ra2/rules/player.yaml

```diff
@@ -74,7 +74,7 @@ Player:
 		Id: medium
 	ProvidesTechPrerequisite@unrestricted:
```

Comment:
You probably want to rename those, too.

Reply:
I didn't rename them because we'd have `Low`, `No Superpowers`, `Unrestricted` which feels like going from 0 to 100 _very quickly_.

I think it'll be best to close this PR and rethink/redo the tech levels as @MustaphaTR mentioned, opinions?

Reply:
If you want to. The current changes make sense to me, but when a major rework of the levels is coming soon anyway, it is indeed best to close this PR.